instrumentation/rocketmq-client/src/test/java/brave/rocketmq/client/RocketMQContainer.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright The OpenZipkin Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package brave.rocketmq.client;
+
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.containers.output.Slf4jLogConsumer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.utility.DockerImageName;
+
+import java.io.IOException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.List;
+
+final class RocketMQContainer extends GenericContainer<RocketMQContainer> {
+  static Logger LOGGER = LoggerFactory.getLogger(RocketMQContainer.class);
+  static final int NAMESERVER_PORT = 9876;
+  static final int BROKER_PORT = 10911;
+
+  RocketMQContainer() {
+    super(DockerImageName.parse("apache/rocketmq:5.3.1"));
```

Comment:
I looked and this is multi-arch now. good! https://hub.docker.com/r/apache/rocketmq/tags